commcare_export/cli.py

```diff
@@ -46,7 +46,9 @@ def main(argv):
     parser.add_argument('--project')
     parser.add_argument('--username')
     parser.add_argument('--password')
-    parser.add_argument('--auth-mode', default='digest', help='Use "session" based auth or "digest" auth.')
+    parser.add_argument('--apikey')
```

Comment:
any reason not to just use the `--password` arg instead of adding a new one?

Reply:
Changed it to us the `--password` arg, and added some help text here:
https://github.com/dimagi/commcare-export/pull/71/commits/f3ce37697c59423070378ee40bd0e9392ce72f1d